pages/projects/[id]/index.js

```diff
@@ -222,11 +235,18 @@ const StyledMaterialsList = styled.ul`
 
 const StyledInstructionsList = styled.ol`
   list-style-position: inside;
+  padding: 0 1rem;
+  margin-bottom: 1rem;
+  list-style-position: inside;
+  padding: 0 1rem 0 1rem;
+  margin-bottom: 1rem;

```

Comment:
padding and margin-bottom are duplicated. 

```suggestion
  padding: 0 1rem;
  margin-bottom: 1rem;
  list-style-position: inside;


```